### ISSUE TITLE
Accessibility support for the blind

### ISSUE THREAD

**Issue by fisher729** [jely2002/youtube-dl-gui#698]
Hi.

I filed this under bug report because accessibility is never a feature.

When I use your great program with NVDA, I cannot do anything with the window once I Alt+Tab away from it and come back. Also, some of the links do not have proper tags, like the login link. It says login disabled and enabled at the same time.

**Comment by jely2002** [comment]
Hi, thanks for the bug report as these things are bugs not features. The alt-tab issue has been resolved and was caused by a bug in the underlying framework (Tauri). See: #688. The fix will be released in the next release v3.1.4 or v3.2.0.

When I went through the app with NVDA I did notice the toggle buttons were behaving in a weird way. I've just now pushed a fix for this, which will also be included in the next release. If you run into something else, don't hesitate to mention it.

**Comment by fisher729** [comment]
I love this. At first I thought my title and descriptions were too vague, and that you would never understand what my issue is. Now that I have hopefully gotten success with this issue, I will be sure to point out any of my accessibility issues going further. Of course, I'd like to see the new version first before I move forward, as this is a major issue. Thanks for such an awesome program.

**Comment by fisher729** [comment]
Hi.

When will the new version be released? It's not very easy to use without this fix for me.

**Comment by jely2002** [comment]
The new version has since been released, so closing this!
